src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbOperations.java

```diff
@@ -34,11 +35,11 @@
 
     <T> T insert(String collectionName, T objectToSave, PartitionKey partitionKey);
 
-    <T> void upsert(T object, PartitionKey partitionKey);
+    <T> void upsert(T object, com.microsoft.azure.documentdb.PartitionKey partitionKey);
```

Comment:
Is the full qualified class name on purpose?

Reply:
keep the full name for now, as we are migrating from old sdk to new sdk. They use the same ClassName. The prefix will be clean up, as well as the old sdk after all done.

Reply:
Sounds good except that I see there are mixed usage of the old SDK and the new SDK. See the above line.

Reply:
Yes, for mini-impact to code, as well as ensuring separate whole work to co-work tasks. If we do the migration in one PR, it will be quit huge. In this way, I and weiping may take different part of Operation like insert/save/find ... etc. After all done, the old sdk will be removed.
The PR do not merge to master tree for now, just one new branch use as developement.